### DISCUSSION TITLE
Cruiser 3.0.40

### DISCUSSION THREAD

**Discussion by devemux86** [devemux86/cruiser#432] [Announcements]
Cruiser 3.0.40 is on [Google Play](https://play.google.com/store/apps/details?id=gr.talent.cruiser):

- https://github.com/devemux86/cruiser/discussions/412

- https://github.com/devemux86/cruiser/discussions/416
(long press direction icon)

- https://github.com/devemux86/cruiser/discussions/411

- https://github.com/devemux86/cruiser/discussions/176#discussioncomment-8270684 improvements

<img src="https://github.com/devemux86/cruiser/assets/3484020/75df8ff5-2b6d-477f-9667-151bbdd508ac" width="50%"/><img src="https://github.com/devemux86/cruiser/assets/3484020/d178d198-914e-491c-ad9b-2785c8d6dfc0" width="50%"/>
<img src="https://github.com/devemux86/cruiser/assets/3484020/ddd320b9-eecd-4b7e-9fc7-5c4511c5a99e" width="50%"/><img src="https://github.com/devemux86/cruiser/assets/3484020/2e18d6fa-4341-4194-8d43-5638f5d10aab" width="50%"/>
![Screenshot_1704959805](https://github.com/devemux86/cruiser/assets/3484020/3a44db88-89fd-4228-81c3-0f95fdf852e5)
![Screenshot_1704959575](https://github.com/devemux86/cruiser/assets/3484020/c771bc7b-cc53-44a7-a3bd-071908aa5bd5)

**Comment by devemux86** [comment]
Cruiser 3.0.39 is on [Google Play](https://play.google.com/store/apps/details?id=gr.talent.cruiser) with various improvements and:

- https://github.com/devemux86/cruiser/discussions/434

**Comment by Ciutnik** [comment]
Really, managing many and long tracks works great, without any hitches. Thank you once again, and BTW two requests for corrections in the Polish language version:

In the track recording menu 
EN: `Clear GPS recording`
should be
PL: `Wyczyść nagranie GPS`
_(currently it is incorrect `Wyraźne nagrywanie GPS`)_

While navigating
EN: `Stop navigation`
should be
PL: `Zatrzymaj nawigację`
_(currently is `Zatrzymać nawigację`, whose form does not match the rest of the translations)_

Thanks! 🙂

**Reply by devemux86** [reply]
@Ciutnik thanks, I will update the texts.

**Comment by devemux86** [comment]
Cruiser 3.0.40 is on [Google Play](https://play.google.com/store/apps/details?id=gr.talent.cruiser) with the fix for this [BRouter report](https://github.com/devemux86/cruiser/discussions/438) and:

- https://github.com/devemux86/cruiser/discussions/176#discussioncomment-8270684 improvements
